packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -53,7 +53,10 @@ function InteractionLayer({
   const [creating, setCreating] = useState(false)
   const svgContext = useContext(SVGContext)
   const canvasRef = useRef()
-  svgContext.canvas = canvasRef.current
+  
+  if (canvasRef.current) {
+    svgContext.canvas = canvasRef.current
+  }
```

Comment:
My attempt at fixing this was almost identical:
```js
svgContext.canvas = canvasRef.current || svgContext.canvas
```

However, I found that this regresses #6490, which this line originally fixed.